doc/OnlineDocs/citing_pyomo.rst

```diff
@@ -4,9 +4,10 @@ Citing Pyomo
 Pyomo
 -----
 
+Bynum, Michael L., Gabriel A. Hackebeil, William E. Hart, Carl D. Laird, Bethany L. Nicholson, John D. Siirola, Jean-Paul Watson, and David L. Woodruff. Pyomo - Optimization Modeling in Python. Springer, 2021.
```

Comment:
Looking at this again, it should probably be ..."Modeling in Python, 3rd Edition. Springer"...